jobs/quicksuggest2bq/tests/test_main.py

```diff
@@ -57,7 +64,7 @@ def test_suggestion_properties_are_properly_parsed(self):
         KintoSuggestion(**SAMPLE_SUGGESTION)
 
     def test_suggestion_download(self, mocked_kinto_client):
-        suggestions = download_suggestions(mocked_kinto_client)
-        assert len(suggestions) == 1
-        assert SAMPLE_SUGGESTION["id"] in suggestions
-        assert suggestions[SAMPLE_SUGGESTION["id"]].title == SAMPLE_SUGGESTION["title"]
+        suggestions = list(download_suggestions(mocked_kinto_client))
+        assert len(suggestions) == 2
+        assert suggestions[0] == KintoSuggestion(**SAMPLE_SUGGESTION)
+        assert suggestions[1] == KintoSuggestion(**SAMPLE_SUGGESTION)
```

Comment:
Can we change the test a bit so that we get a slightly different `SAMPLE_SUGGESTION` here? Just to make sure we're returning different records.

Reply:
Good point. 👍